homework-3/src/test/java/RandomString.java

```diff
@@ -0,0 +1,31 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class RandomString {
+    public static void main(String[] args) {
+        String[] stringsList = {"Hello World", "Have a great day", "Angel", "Random String", "Word Game"};
+        List<String> filteredStringList = new ArrayList<String>();
```

Comment:
```suggestion
        List<String> filteredStringList = new ArrayList<>();
```

Reply:
this called diamond syntax.
Purpose of the diamond operator is to simplify the use of generics when creating an object.
It avoids unchecked warnings in a program as well as reducing generic verbosity by not requiring explicit duplicate specification of parameter types.